README.md

```diff
@@ -9,6 +9,7 @@ The latest version is Version 1.3. To install or update Math Symbols Input:
 
 1. Download the [installer for the latest version](https://github.com/knrafto/MathSymbolsInput/releases/latest/download/MathSymbolsInput.pkg).
 2. Open the installer and continue through the installation steps.
+3. A reboot may be required
```

Comment:
```suggestion
3. Log out and log back in (or reboot).
```

Thanks! I think logging out should work? It's only a little faster than rebooting though.